alpha/patterns/addon/examples/dashboard-operator/.gitignore

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
I am wondering if we should move the `examples` outside the `alpha/patterns/...` to maintain separation between the packages user import and the examples that they refer. This will also help tools because tools like `dep` considers examples to be part of the imported code.